addons/a11y/src/components/Report/index.js

```diff
@@ -16,52 +16,25 @@ const styles = {
     textAlign: 'center',
     textTransform: 'uppercase',
   },
-  rerunButtonContainer: {
-    textAlign: 'center',
-    margin: 10,
-  },
-  rerunButton: {
-    textTransform: 'uppercase',
-    letterSpacing: 1,
-    border: '1px solid rgb(193, 193, 193)',
-    borderRadius: 2,
-    padding: 5,
-  },
 };
 
-function RerunButton() {
-  function onRerunClick() {
-    const channel = addons.getChannel();
-    channel.emit('addon:a11y:rerun');
-  }
-  return (
-    <div style={styles.rerunButtonContainer}>
-      <button onClick={onRerunClick} style={styles.rerunButton}>
-        Re-run tests
-      </button>
-    </div>
-  );
+function onRerunClick() {
+  const channel = addons.getChannel();
+  channel.emit('addon:a11y:rerun');
 }
 
-function Report({ items, empty, passes }) {
-  if (items.length) {
-    return (
-      <div>
-        <div style={styles.container}>
-          {items.map(item => <Item passes={passes} item={item} key={item.id} />)}
-        </div>
-        <RerunButton />
+const Report = ({ items, empty, passes }) => (
+  <Fragment>
+    {items.length ? (
+      <div style={styles.container}>
+        {items.map(item => <Item passes={passes} item={item} key={item.id} />)}
       </div>
-    );
-  }
-
-  return (
-    <div>
+    ) : (
       <span style={styles.empty}>{empty}</span>
-      <RerunButton />
-    </div>
-  );
-}
+    )}
+    <RerunButton onClick={onRerunClick}>RE-RUN</RerunButton>
```

Comment:
This change is a regression from "Re-run tests" to "RE-RUN". "RE-RUN" doesn't describe the functionality of the button in full and is also in all-caps; this should be achieved with CSS instead of written out in the HTML.